real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/AnalyticsPanel.java

```diff
@@ -9,7 +9,8 @@
 	private static final long serialVersionUID = 7017667286491619492L;
 
 	private DataPanel dataPanel;
-	private GraphPanel graphPanel;
+//	private GraphPanel graphPanel;
+	private BuggyStatusPanel graphPanel;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'graphPanel' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2343306785/issues/source?bid=2938665&fileBranchId=3212596#l13)